cyhy/core/geoloc.py

```diff
@@ -39,3 +41,22 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_cidr(self, cidr):
+        restricted_ips = IPSet()
+        for ip in cidr:
+            try:
+                response = self.__reader.city(str(ip))
+                if response.country.name in RESTRICTED_COUNTRIES:
+                    restricted_ips.add(ip)
+            except AddressNotFoundError:
+                print >> sys.stderr, "CIDR %s not found in geolocation database" % cidr
+        # to avoid printing each and every ip in a cidr that is found
+        # this is only printing the cidr in which one or more addresses are found
+        # that correspond to a restricted country
```

Comment:
This comment is a bit misleading since we are now printing the list of CIDRs representing every restricted IP found, not "the CIDRs in which one or more addresses are [restricted]".  Can you please clean up the wording here?

Reply:
Ahh right, I changed the functionality without the comment. Thanks

Reply:
Good point- I'm fine with removing that comment.